src/library/Uno.Material/Styles/Controls/v1/DatePicker.xaml

```diff
@@ -108,7 +108,7 @@
 										  BorderThickness="{TemplateBinding BorderThickness}"
 										  CornerRadius="{TemplateBinding CornerRadius}"
 										  Elevation="{StaticResource MaterialDatePickerFlyoutElevation}"
-										  ShadowColor="Black">
+										  ShadowColor="{StaticResource MaterialShadowColor}">
```

Comment:
Does MaterialShadowColor exist in the v1 material color palette

Reply:
Basically we should have a `MaterialShadowColor` in the v1 palette and all v1 styles should reference that one. The v2 styles should reference a `ShadowColor` in the SharedColorPalette for v2